INPTPZ1/Program.cs

```diff
@@ -1,264 +1,175 @@
+            polynome.Coeficients.Add(ComplexNumber.Zero);
+            polynome.Coeficients.Add(new ComplexNumber() { Re = 1 });
+            return polynome;
+        }
+
+        private static void CheckComplexNumber(ComplexNumber complexNumber)
+        {
+            if (complexNumber.Re == 0)
+                complexNumber.Re = 0.0001;
+            if (complexNumber.Im == 0)
+                complexNumber.Im = 0.0001;
+        }
+
+        private static void MakeImage(int numberOfRoots, int iteration, int xCoordinate, int yCoordinate)
+        {
+            Color color = colors[numberOfRoots % colors.Length];
+            int red = Math.Min(Math.Max(0, color.R - iteration * 2), 255);
+            int blue = Math.Min(Math.Max(0, color.B - iteration * 2), 255);
+            int green = Math.Min(Math.Max(0, color.G - iteration * 2), 255);
+            color = Color.FromArgb(red, green,blue);
+            bitmapPicture.SetPixel(yCoordinate, xCoordinate, color);
+        }
+
+        private static bool CheckArguments(string input)
+        {
+            
+            return true;
+        }
+
+        static void Main(string[] args)
+        {
+            try
+            {
+                int inputNumber = int.Parse(args[0]);
+                Initialization(inputNumber, inputNumber);
+            }
+            catch (ArgumentNullException ex)
+            {
+                Console.WriteLine("ArgumentNullException");
+            }
+            catch (ArgumentException ex)
+            {
+                Console.WriteLine("ArgumentException");
+            }
+            catch (FormatException ex)
+            {
+                Console.WriteLine("FormatException");
+            }
+
+
+            try
+            {
+                for (int xCoordinate = 0; xCoordinate < width; xCoordinate++)
+                {
+                    for (int yCoordinate = 0; yCoordinate < height; yCoordinate++)
+                    {
+                        ComplexNumber complexNumber = new ComplexNumber()
+                        {
+                            Re = xMin + yCoordinate * xStep,
+                            Im = yMin + xCoordinate * yStep
+                        };
+
+                        CheckComplexNumber(complexNumber);
+
+                        int iteration = 0;
+                        FinderSolutionByNewtonIteration(ref iteration, ref complexNumber);
+
+                        var numberOfRoots = 0;
+                        FinderSolutionRoot(ref numberOfRoots, complexNumber);
+
+                        MakeImage(numberOfRoots, iteration, xCoordinate, yCoordinate);
+                    }
+                }
+                bitmapPicture.Save("../../../out.png");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Exception");
+            }
+            
+
+        }
+
+        private static void FinderSolutionRoot(ref int numberOfRoots, ComplexNumber complexNumber)

```

Comment:
numberOfRoots - invalid name; and order of parameters should be reversed - first input arguments then output